src/components/Detail/Detail.spec.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import Detail from './Detail'
+
+it('should render a Detail component with the default prop values', () => {
+  const detail = mount(<Detail />)
+
+  expect(detail.prop('cost')).toEqual(0)
```

Comment:
For me, this is trivial. When the component are connected with redux, this `expects` will be removed

Reply:
Yea, this test is useless but it would be the same than using snapshots because currently we are having an issue when we test with snapshots components that uses material ui, it's an issue with the new files created. I couldn't solve that error so that's why I added this test.